pkg/catalog/outbound_traffic_policies.go

```diff
@@ -12,6 +17,24 @@ import (
 	"github.com/openservicemesh/osm/pkg/trafficpolicy"
 )
 
+// Retry Policy
+const (
+	// RetryOnAnnotation is the annotation used to specify what to attempt a retry on
+	RetryOnAnnotation = "openservicemesh.io/retrypolicy/retry-on"
+
+	// NumRetriesAnnotation is the annotation used to specify the number of retries
+	NumRetriesAnnotation = "openservicemesh.io/retrypolicy/num-retries"
+
+	// PerTryTimeoutAnnotation is the annotation used to specify the timeout per retry attempt
+	PerTryTimeoutAnnotation = "openservicemesh.io/retrypolicy/per-try-timeout"
+)
+
+var (
+	// TODO: determine what to retry-on
```

Comment:
Could you open an issue to track this since it seems like we haven't established yet what to retry on, which is important?

Reply:
https://github.com/openservicemesh/osm/issues/4031